doc/joss/paper.md

```diff
@@ -0,0 +1,168 @@
+---
+title: 'Ginkgo: A high performance numerical linear algebra library'
+tags:
+  - linear-algebra
+  - hpc
+  - cuda
+  - modern-c++
+  - hip
+  - spmv
+authors:
+  - name: Hartwig Anzt
+    orcid: 0000-0003-2177-952X
+    affiliation: "1, 2" # (Multiple affiliations must be quoted)
+  - name: Terry Cojean
+    orcid: 0000-0002-1560-921X
+    affiliation: 1 
+  - name: Yen-Chen Chen 
+    affiliation: 4 
+  - name: Goran Flegar
+    orcid: 0000-0002-4154-0420
+    affiliation: 3 
+  - name: Fritz G{\"o}bel
+    affiliation: 1 
+  - name: Thomas Gr{\"u}tzmacher
+    orcid: 0000-0001-9346-2981
+    affiliation: 1
+  - name: Pratik Nayak 
+    orcid: 0000-0002-7961-1159
+    affiliation: 1 
+  - name: Tobias Ribizel 
+    affiliation: 1 
+    orcid: 0000-0003-3023-1849
+  - name: Yu-Hsiang Tsai 
+    orcid: 0000-0001-5229-3739
+    affiliation: 1 
+affiliations:
+ - name: Karlsruhe Institute of Technology
+   index: 1
+ - name: Innovative Computing Laboratory, University of Tennessee, Knoxville 
+   index: 2
+ - name: University of Jaume I 
+   index: 3
+ - name: The University of Tokyo 
+   index: 4
+date: 6th March, 2020.
+bibliography: paper.bib
+
+---
+
+# Summary
+
+Ginkgo is a production-ready sparse linear (operator) algebra library for high
+performance computing on GPU-centric architectures with a high level of
+performance portability and focuses on software sustainability. 
+
+The library focuses on solving sparse linear systems and accommodates a large variety
+of matrix formats, state-of-the-art iterative (Krylov) solvers and preconditioners, 
+which make the library suitable for a variety of scientific applications. Ginkgo
+supports many architectures such as multi-threaded CPU, NVIDIA GPUs, and AMD GPUs.
+The heavy use of modern C++ 11 features simplifies the addition of new executor
```

Comment:
This statement might be considered contradictory, as C++11 is now already a few years old. The main reason why we want C++11 compatibility is that not everybody has the most up-to-date compilers, right? Maybe we should state that somewhere? We could then just write modern C++ here.